templates/bridge-exporter.yaml

```diff
@@ -149,9 +149,14 @@ Resources:
         - Namespace: 'aws:autoscaling:launchconfiguration'
           OptionName: SecurityGroups
           Value: !Join
-            - '-'
+            - ''
```

Comment:
SG values should not be hard coded, instead it should be:

```
        - Namespace: 'aws:autoscaling:launchconfiguration'
          OptionName: SecurityGroups
          Value: !Join
            - ''
            - - !Ref 'AWS::StackName'
              - '-'
              - AWSEC2SecurityGroup
              - ','
              - !ImportValue 'bridge-BastianSecurityGroup'
              - ','
              - !ImportValue 'bridge-VpnSecurityGroup'
```

The reason why ImportValue doesn't work is because the SG name cannot be exported.  The cloudformation does not provide access to the SG name.  It only provides access to the SG Id while EB wants the SG name.